libnet/Makefile.am.common

```diff
@@ -5,7 +5,6 @@
 # All rights reserved.
```

Comment:
In this case, the `*~` files are actually Emacs backup files. Completely inappropriate here!